packages/client/src/components/styledComponents/Profile/Profile.tsx

```diff
@@ -1,14 +1,43 @@
 import styled from 'styled-components'
 import variables from '@/Variables.module.scss'
+import { DarkModeType, ProfileProps } from '@/types'
 
```

Comment:
```suggestion
import { Button } from '@/components'
```
Пояснение в след комментарии